### PR TITLE
neuralprophet 0.8.0

### PR DESCRIPTION
**Pull request by ifitchet**
neuralprophet 0.8.0

**Destination channel:** defaults

### Links

- [PKG-4657]
- dev_url (pypi): https://github.com/ourownstory/neural_prophet/tree/0.8.0
- conda-forge: https://github.com/conda-forge/neuralprophet-feedstock/blob/main/recipe/meta.yaml
- pypi: https://pypi.org/project/neuralprophet/0.8.0
- pypi inspector: https://inspector.pypi.io/project/neuralprophet/0.8.0

### Explanation of changes:

- updated dependencies for 0.8.0
- updated `about:` with data from pypi

[PKG-4657]: https://anaconda.atlassian.net/browse/PKG-4657?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: abs.yaml
Comment (ViridianMelody) [inline abs.yaml:1]: Make sure to remove the staging channel before merging.

**Review by ViridianMelody — APPROVED** [review]
Approved pending `pytorch-lightning` upload and staging channel removal.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**
